notebooks/FAISS_with_🤗_datasets_and_CLIP.ipynb

```diff
@@ -0,0 +1,10919 @@
+{
```

Comment:
<p>"we will <a href="https://huggingface.co/openai/clip-vit-base-patch16" target="_blank">CLIP model</a> " =&gt; "we will use <a href="https://huggingface.co/openai/clip-vit-base-patch16" target="_blank">CLIP model</a>"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>